src/config.js

```diff
@@ -33,7 +33,8 @@ let defaults = {
   PUBCHEM_LINK_BASE_URL: 'https://pubchem.ncbi.nlm.nih.gov/compound/',
   PUBCHEM_CACHE_SIZE: DEFAULT_CACHE_SIZE,
   AGGREGATE_CACHE_SIZE: DEFAULT_CACHE_SIZE,
-  MAX_SEARCH_SIZE: 50
+  MAX_SEARCH_SIZE: 50,
+  BIOPAX_CONVERTER_URL: 'http://localhost:9090/FactoidToBiopaxServer/ConvertToOwl'
```

Comment:
@metincansiper @jvwong Could the BIOPAX_CONVERTER server be hosted somewhere with a public URL so everyone doesn't have to host locally to test?

Reply:
Where would it be hosted? If needed we have a google cloud machine hosted at http://causalpath.org/.

Reply:
@metincansiper If possible, it might be better for you guys to host it -- at least for now.